docs/design/api.rst

```diff
@@ -113,7 +111,7 @@ This makes a separate “You’re subscribed to [product]” email unnecessary.
 Subscription charge succeeded
 +++++++++++++++++++++++++++++
 
-The reception of email receipt indicates that the charge was succeeded.
+The reception of email receipt already indicates that the charge was succeeded. No need to send a separate email.
```

Comment:
So we'll be raising an event "subscription charge succeeded event" in the code. That will be interpreted as "send a receipt". If we word it "When a subscription charge succeeds a receipt is sent." it might make sense.